backend/src/main/kotlin/com/group7/artshare/entity/ArtItem.kt

```diff
@@ -39,10 +40,9 @@ class ArtItem{
     @Column
     var onAuction: Boolean = false
 
-    @OneToOne(cascade = [CascadeType.ALL])
-    @JoinColumn(name = "auction", referencedColumnName = "id")
+    @OneToMany(orphanRemoval = true, cascade = [CascadeType.ALL])
     @JsonIdentityInfo(generator = ObjectIdGenerators.PropertyGenerator::class, property = "id")
-    var auction: Auction? = null
+    var bids: MutableList<Bid> = mutableListOf()
```

Comment:
I think storing only the highest bid is enough. Others are not required.

Reply:
Updated the code according to this. Thanks for pointing out.